tests/neg/missing-implicit4.check

```diff
@@ -16,12 +16,7 @@
    |
    |    import instances.traverseList
    |
--- Error: tests/neg/missing-implicit4.scala:20:42 ----------------------------------------------------------------------
+-- [E008] Not Found Error: tests/neg/missing-implicit4.scala:20:18 -----------------------------------------------------
 20 |    List(1, 2, 3).traverse(x => Option(x)) // error
-   |                                          ^
-   |no implicit argument of type Zip[Option] was found for an implicit parameter of method traverse in trait Traverse
-   |
-   |The following import might fix the problem:
-   |
-   |  import instances.zipOption
-   |
+   |    ^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
same here, tests should maintain the same error message, so the extension methods should be manually desugared